core/unit_test/TestNumericTraits.hpp

```diff
@@ -14,12 +14,7 @@
 //
 //@HEADER
 
-#include <gtest/gtest.h>
-
-#include <Kokkos_Core.hpp>
-#include <type_traits>
-#include <limits>
-#include "Kokkos_NumericTraits.hpp"
+#include <Kokkos_Macros.hpp>
```

Comment:
This might be a stupid question but why don't we move the diagnostic pragmas to `Kokkos_complex.hpp` itself ?

Reply:
`Kokkos_Complex.hpp` doesn't contain `long double`.